packages/ssx-server/src/middlewares/express/middleware.ts

```diff
@@ -87,28 +87,29 @@ export const ssxMiddleware = (ssx: SSXServer) => {
     };
 
     if (req.session?.siwe) {
+      const { signature, siwe, daoLogin, nonce } = req.session;
+      let siweMessageVerify;
       try {
-        const { signature, siwe, daoLogin, nonce } = req.session;
-        const { success: verified, data } = await new SiweMessage(siwe).verify(
+        siweMessageVerify = await new SiweMessage(siwe).verify(
           { signature, nonce },
           {
             verificationFallback: daoLogin ? SiweGnosisVerify : null,
             provider: ssx.provider,
           },
         );
-
-        if (verified) {
-          req.ssx = {
-            ...req.ssx,
-            siwe: data,
-            verified,
-            userId: `did:pkh:eip155:${siwe.chainId}:${siwe.address}`,
-          };
-        } else {
-          req.session.destroy(() => next());
-        }
       } catch (error) {
-        // ignore errors? Log them?
+        console.error(error);
```

Comment:
We don't want to error here, it will show up on requests for any express server consuming this middleware. We will need to consider error configuration + logging later